config/config.json

```diff
@@ -0,0 +1,25 @@
+{
+  "monitoringCluster": "mad",
+  "logDirectory": "logs",
+  "pipelinesDirectory": "/opt/mad/config/pipelines",
+  "httpHost": "0.0.0.0",
+  "httpPort": 7090,
+  "httpHealthCheckPath": "/ping",
+  "httpStatusPath": "/status.json",
+  "jvmMetricsCollectionInterval": "PT.5S",
+  "akkaConfiguration": {
+    "akka": {
+      "loggers": [
+        "akka.event.slf4j.Slf4jLogger"
+      ],
+      "loglevel": "DEBUG",
+      "stdout-loglevel": "DEBUG",
+      "logging-filter": "akka.event.slf4j.Slf4jLoggingFilter",
+      "actor": {
+        "debug": {
+          "unhandled": "on"
+        }
+      }
+    }
+  }
+}
```

Comment:
So I didn't get all of it over the phone, but you mentioned that the docker image was being created for development and that implies the config is for development as well. Is this correct?

Do you intend the same Dockerfile to build the test/dev image and the production image? (documentation and file location are considerations here)

Similarly, is the config directory intended for test/dev only or for production? (some consideration for future file structure is likely needed here)